src/profile-logic/process-profile.js

```diff
@@ -223,26 +248,117 @@ function processThread(thread, libs) {
             );
           }
           fileName = stringTable.indexForString(scriptURI);
-          lineNumber = jsMatch[3] | 0;
+          lineNumber = parseInt(jsMatch[3], 10);
         }
       }
     }
     funcIndex = funcTable.length;
-    addFunc(
-      funcNameIndex,
-      resourceIndex,
-      addressRelativeToLib,
-      isJS,
-      fileName,
-      lineNumber
-    );
+    {
+      // Add the function to the funcTable.
```

Comment:
Mainly to not add another variable name to the large function, and because the word `index` being so generic. I liked how the original code read.